core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1434,6 +1434,9 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean, dynamicConfigO
   val zkEnableSecureAcls: Boolean = getBoolean(KafkaConfig.ZkEnableSecureAclsProp)
   val zkMaxInFlightRequests: Int = getInt(KafkaConfig.ZkMaxInFlightRequestsProp)
 
+  private val _tieredKafkaConfig = new RemoteLogManagerConfig(this)
```

Comment:
Why is this prefixed with `tiered` while the class name is prefixed with `Remote`?

Reply:
I had a hard time naming this variable. Would `_remoteLogManagerConfig` be better?